repeat.lua

```diff
@@ -33,6 +34,8 @@ Arguments:
     ``...`` specifies the command to be run
 ``-cancel NAME``
     cancels the repetition with the name NAME
+``list``
```

Comment:
I'd suggest making it `-list` for consistency with the other arguments (although I'd be okay with it taking both `list` and `-list`)

Reply:
I've changed it to `-list` but also slightly changed it's semantic: it doesn't make `repeat` ignore everything else anymore, instead, the list is printed after a command is added or canceled (if any).